src/content/en/fundamentals/performance/webpack/decrease-frontend-size.md

```diff
@@ -13,7 +13,27 @@ description: How to use webpack to make your app as small as possible
 One of the first things to do when you’re optimizing an application is to make it as small as
 possible. Here’s how to do this with webpack.
 
-## Enable minification
+## Use the production mode (webpack 4 only) {: #use-the-production-mode }
+
+Webpack 4 introduced the new `mode` flag. You could set this flag to `'development'` or
+`'production'` to hint webpack that you’re building the application for a specific environment:
+
+    // webpack.config.js
+    module.exports = {
+      mode: 'production',
+    };
+
+Make sure to enable the `production` mode when you’re building your app for production.
+This will make webpack apply optimizations like minification, removal of development-only code
+in libraries, [and more](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a).
+
```

Comment:
What do you think about this paragraph including a bullet list of the optimizations that are applied through `production` mode, each linked to where you otherwise manually enable them from this guide?

Reply:
IDK, have no opinion here. What’s the benefit/use of this?

Reply:
I think the benefit is making it clear to the reader what the mode does (in case just using `production` without anything else is enough), but we don't need to do that.

Reply:
> I think the benefit is making it clear to the reader what the mode does

That’s a good point! I would probably link to the webpack docs instead once they are updated – it seems slightly weird for me to link to a section which has a “Skip this section if you’re using the production mode” notice. What do you think?